packages/terra-alert/tests/wdio/alert-spec.js

```diff
@@ -62,12 +62,25 @@ Terra.describeViewports('Alert', ['tiny', 'large'], () => {
   });
 
   describe('Critical alert notification banner', () => {
-    it('alert content is focused when rendered with an action element', () => {
+    it('alert content is focused when rendered with an action element and dismiss', () => {
       browser.url('/raw/tests/cerner-terra-core-docs/alert/custom-prop-alert');
-
       browser.keys(['Tab', 'Tab', 'Tab', 'Tab', 'Enter']);
 
-      Terra.validates.element('alert focused');
```

Comment:
Can these obsolete reference screenshots be removed?

Reply:
Thanks for catching that, removed here https://github.com/cerner/terra-core/pull/3823/commits/5ff0e554ca3d159a06b6a78b3db69905d321d860